src/Stubs/PHPStanStubLoader.php

```diff
@@ -44,7 +44,7 @@ public function loadStubs(): void
             }
 
             foreach (self::STUBS as $stub) {
-                $path = sprintf('phar://%s/phpstan/phpstan/phpstan.phar/stubs/runtime/%s', $vendorPath, $stub);
+                $path = sprintf('%s/phpstan/phpstan-extracted/stubs/runtime/%s', $vendorPath, $stub);
```

Comment:
this is to handle phpstan's stubs got from phpstan-extracted instead of the .phar.